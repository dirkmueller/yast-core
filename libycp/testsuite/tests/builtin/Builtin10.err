Parsed:
----------------------------------------------------------------------
{
    // list MyList
    // list NewList
    // filename: "tests/builtin/Builtin10.ycp"
    list MyList = ["Austria", "Belgium", "China", `D_Nation, "England", "France", "Greece"];
    list NewList = [];
    NewList = prepend (MyList, .Hungary);
    return ((size (MyList) == 7) && (/* any -> path */select (NewList, 0, nil) == .Hungary));
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
    // filename: "tests/builtin/Builtin10.ycp"
    return "test case prepend for list finished";
}
----------------------------------------------------------------------
