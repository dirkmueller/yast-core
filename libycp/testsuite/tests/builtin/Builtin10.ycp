
# ---------------------------------------------------------
#
#  Filename:    Builtin10.ycp
#
#  Purpose:     test cases for YCP test
#
#  Creator:     jd@suse.de
#
#  Maintainer:  jd@suse.de
#
# ---------------------------------------------------------

// Creates a new list that is identical to the list but has
// the value  prepended as additional element.

{
     list  MyList = [    "Austria",
                          "Belgium",
                          "China",
                          `D_Nation,
                          "England",
                          "France",
                          "Greece" ];
     list  NewList = [];

//   "one entry within the list is added (prepend)"
     NewList = prepend( MyList, .Hungary );
     return( ( size( MyList ) == 7 ) &&
             ( (path)select( NewList, 0, nil ) == .Hungary ) );
}

{
    return( "test case prepend for list finished" );
}

