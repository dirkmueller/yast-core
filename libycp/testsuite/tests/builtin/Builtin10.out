(true)
("test case prepend for list finished")
